pom.xml

```diff
@@ -56,8 +56,6 @@
         <guava.version>29.0-jre</guava.version>
         <gson.version>2.8.5</gson.version>
         <hapi.version>4.1.0</hapi.version>
-        <httpclient.cache.version>4.5.5</httpclient.cache.version>
-        <httpclient.version>4.5.12</httpclient.version><!-- keep in sync with both Camel and Quarkus -->
```

Comment:
Is Quarkus managing these now?

Reply:
https://github.com/quarkusio/quarkus/blob/1.9.0.CR1/bom/application/pom.xml#L2846-L2855

Reply:
Haha, it was even me adding httpmime https://github.com/quarkusio/quarkus/blame/1.9.0.CR1/bom/application/pom.xml#L2851-L2855
Thanks for reminding!